src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -895,20 +895,25 @@ private static async Task<DurableOrchestrationStatus> GetDurableOrchestrationSta
                                     ConvertOutputToJToken(historyItem, showHistoryOutput);
                                     break;
                                 case EventType.ExecutionTerminated:
-                                    historyItem.Remove("Input");
+
                                     break;
                                 case EventType.TimerFired:
                                     historyItem.Remove("TimerId");
                                     break;
                                 case EventType.EventRaised:
-                                    historyItem.Remove("Input");
+
```

Comment:
nit: please remove the extra newlines before the `break` statement here and in the above cases. That way your change doesn't impact the original formatting pattern of the code.